users/templates/users/password_reset_form.html

```diff
@@ -0,0 +1,18 @@
+{% extends 'base.html' %}
+{% load crispy_forms_tags %}
+
+{% block content%}
+    <div class="container text-center mt-2 container-reset">
+        <h1>Password Reset Form</h1>
+        <p style="width: 60%; margin: 0 auto">
+            Please enter your new password so we can verify.
+        </p>
+        <br />
+        <form action="" method="POST">
+            {% csrf_token %}
+            {{ field.errors }}
```

Comment:
This won't work. You are using field.errors not "non_field_errors" as was requested. Please check registration and just copy that code for non_field_errors here. 
Also I don't even know if field.errors exist.